src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -5489,7 +5490,63 @@ public void testListNodes() throws Exception {
       assertThat(plugin.name(), allOf(notNullValue(), not("")));
     }
   }
-  
+
+  @Test
+  public void testInspectNode() throws Exception {
```

Comment:
@mattnworb Do we want to add integration tests like these at all anymore? Seems like if we have a unit test we can skip this.